python/python/glide/async_commands/bitmap.py

```diff
@@ -24,7 +24,10 @@ class BitmapIndexType(Enum):
 
 class OffsetOptions:
     def __init__(
-        self, start: int, end: int, index_type: Optional[BitmapIndexType] = None
+        self,
+        start: Optional[int] = None,
+        end: Optional[int] = None,
```

Comment:
Please update docs (lines 40-41)

Reply:
It is possible to define `end`, but skip `start`. The command won't fail in that case, but will return incorrect data.
Probably we need to create multiple factory methods to protect us from that.

Reply:
As another option - we can keep `start` mandatory. If user wants to omit `start` - `bitpos` should be callsed without options

Reply:
Agree with the latest comment: keep `start` mandatory, and when user want to omit both `start` and `end`, they can just omit the entire `OffsetOptions`, as itself is an optoinal parameter to the actual command fucntions, like `bitpos()`. We probably want to make the same change to Node as well, to keep consistency.